fix(nd): Updated active waypoint to be left aligned

MrJigs7 opened the pull request:
Updated active waypoint to be left aligned
Updated text X position

<!-- ⚠⚠ Do not delete this pull request template! ⚠⚠ -->
<!-- Pull requests that do not follow this template are likely to be ignored. -->

<!-- Add the issues this PR fixes here. If no issues are related to this PR, then this line can be removed. -->
<!-- Add further issues with a full "Fixes #[issue_no]" line to ensure GitHub closes each one when the PR is merged. -->
Fixes #9472

## Summary of Changes
Updated active waypoint to be left aligned
Updated text X position to correctly display active waypoint on ND. 

## Screenshots (if necessary)
![Before Change](https://github.com/user-attachments/assets/15b90295-c696-4518-9722-dcfe9135e4ca)
![3 character waypoint](https://github.com/user-attachments/assets/30635ce6-511a-4cb0-b5c5-2bc19997c972)
![5 character waypoint](https://github.com/user-attachments/assets/d142fd6c-460b-4ed9-a269-e8325bef2bfd)
![6 character waypoint](https://github.com/user-attachments/assets/73e99ace-096b-478f-a904-992190632654)
![7 character waypoint](https://github.com/user-attachments/assets/a0b591e8-a8a6-49dd-92b7-0dcb54309bfa)



## References
<!-- You should be making changes based on some kind of a reference (manuals, videos, IRL photos). P3D/xplane/fsx references will only be accepted if we believe that one cannot reasonably obtain a better source. Please post screenshots of the references you used. Ask around in the discord for how to find references for what you are working on. Exceptions will probably be made for IRL A320 pilots and engineers. -->

![RealWorld A380 ND](https://github.com/user-attachments/assets/08d3f4a2-bd81-48cc-bd7b-9a0eb51f9890)

<!-- If you are making a pull request related to the MCDU, please make sure you are ONLY referencing the Honeywell Pegasus Step 1A (Rev 0), 2009 edition manual. -->
<!-- If you do not have this manual, please ask on our discord for assistance -->

## Additional context
<!-- Add any other context about the pull request here. -->

<!-- You may optionally provide your discord username, so that we may contact you directly about the issue. -->
Discord username (if different from GitHub):MrJigs.

## Testing instructions
<!-- Detail how this PR should be tested by QA. Try to list important items that need checking, either directly changed by this PR or that could be affected by it -->
1. Load a flight plan, insert some waypoints - make sure not in PPOS
2. Check active waypoint is now left aligned
3. Check waypoints of various character lengths to check correct positioning and make sure no fonts clip with other ND details
4. There should be no change on the A32NX

<!-- DO NOT DELETE THIS -->
## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page

tracernz commented:
Do you have any A320 family references? You are changing it for both planes here.

CaptCrystal commented:
![image](https://github.com/user-attachments/assets/d643f6e8-2ad5-4e42-911f-80bf3844d991)
@tracernz I don't know if this was followed for this pr, but here is a reference for the a320neo.

heclak commented:
I think this is wrong for the A320 Neo. Based on this reference with a three char waypoint name, I believe it is right aligned on the A320.
![image](https://github.com/user-attachments/assets/e0181eb0-8a77-4791-93b5-95ddfe34a1f0)

https://youtu.be/Y1yQziRUFNY?si=RG2w7aXdU0aBFEt1 (3:06)

tracernz commented:
An appropriate CSS class (with transform rather than the SVG x/y attrs) + different CSS in the ND stylesheet for the A380 might be the easiest way to solve it.

MrJigs7 commented:
> An appropriate CSS class (with transform rather than the SVG x/y attrs) + different CSS for that class in the ND stylesheet for the A380X might be the easiest way to solve it.

I am still learning CSS so and my way round the repo so unfortunately I am not 100% sure on how exactly to do this but if a similar change or reference could be provided, I could take a look

MrJigs7 commented:
Had a further look into this, not sure if I am on the right track,  in the fbw-common ND is it better to change to render all of the ToWaypointIndicator class using transform or just the toWptIdentValue ?

MrJigs7 commented:
I've updated this PR so that the waypoint indicator is now rendered with a CSS class using transform and added styling requirements to the A380X ND stylesheet. Tested on both A320 and A380 to check alignment is correct.

tracernz commented on fbw-a380x/src/systems/instruments/src/ND/style.scss:
```diff
@@ -34,3 +34,9 @@
+  text-align: start;
+  text-anchor: start;
+  transform: translateX(-130px) translateY(0);
+}
```
Missing linefeed (that's what the red symbol means).

MrJigs7 replied:
Fixed now

tracernz commented:
Can you add that there should be no change in the A32NX in test instructions please?

MrJigs7 commented:
> Can you add that there should be no change in the A32NX in test instructions please? Your discord username above did not work to ping you in the QA channel 😛 .

Done 😊 corrected Discord name as well

Jonny23787 commented:
<!-- !! DO NOT REMOVE THIS LINE !! - QUALITY ASSURANCE REPORT -->
<!-- Refer to the https://docs.flybywiresim.com/dev-corner/qa-process/#reporting-on-tests for details about this template -->

<!-- Please use one or the other depending on your Discord Role -->
Quality Assurance Trainee Report

Discord Username  : jonny_23 <!-- Use your username, not your display name! -->
Object of testing : #9547  <!-- PR Number -->
Aircraft          : A380X <!-- Choose only one -->
Tier of Testing   : 1
Date              : 05/03/2025 <!-- Date of test in DD/MM/YYYY format -->

Testing Process:

1. Spawned at YSSY
2. Entered departure and flight plan in FMS
3. Checked that waypoint was left aligned on ND
4. Checked various other waypoints were left aligned (Not all screenshotted)
5. Confirmed that A32NX was unchanged 

Testing Results: 
Passed <!-- Required section --> 

Negatives:       
<!-- Ommit this section if none are found -->

Conclusions:
Passed 
<!-- Ommit section if nothing to add -->

Media:
![Screenshot 2025-02-28 201319](https://github.com/user-attachments/assets/b6288ce5-5c52-4e61-8ff0-a82f55934752)
![Screenshot 2025-03-05 154057](https://github.com/user-attachments/assets/bd6d7a9b-b59a-41c6-9718-7f27670c19b9)
![Screenshot 2025-02-28 201554](https://github.com/user-attachments/assets/2d02c8dc-3d66-4d75-bad1-2856f85bb969)


<!-- Ommit this section if no media to add -->
